client/plots/geneORA.js

```diff
@@ -0,0 +1,216 @@
+import * as client from '../src/client'
+import { renderTable } from '../dom/table'
+import * as d3axis from 'd3-axis'
+import { controlsInit } from './controls'
+import { select as d3select } from 'd3-selection'
+import { getCompInit, copyMerge } from '#rx'
+import { dofetch3 } from '#common/dofetch'
+import { scaleLog, scaleLinear } from 'd3-scale'
+import { d3lasso } from '../common/lasso'
+import { downloadTable } from '../dom/table'
```

Comment:
combine with line 2

Reply:
I realized, this code was making a lot of unnecessary imports (for e.g d3 stuff). I just deleted those lines in my latest commit. I suppose then this comment is no longer relevant?